webapp/channels/src/components/announcement_bar/default_announcement_bar/announcement_bar.tsx

```diff
@@ -164,7 +164,7 @@ export default class AnnouncementBar extends React.PureComponent<Props, State> {
 
         const announcementIcon = () => {
             return this.props.showLinkAsButton &&
-            (this.props.showCloseButton ? <div className='content__icon'>{'\uF5D6'}</div> : <div className='content__icon'>{'\uF02A'}</div>);
+            (this.props.showCloseButton ? <i className='icon icon-alert-circle-outline'/> : <i className='icon icon-alert-outline'/>);
```

Comment:
This is the actual fix for the issue. I haven't found when exactly the code point for the emojis changed, but it seems to have been at least a year ago